en/micro-integrator/docs/setup/brokers/configure-with-rabbitMQ.md

```diff
@@ -22,9 +22,32 @@ parameter.username = "guest"
 parameter.password = "guest"
 ```
 
+## Enabling the RabbitMQ Sender
+
+Uncomment the following parameters in the `deployment.toml` file to enable the RabbitMQ sender.
+
+```toml
+[transport.rabbitmq]
+sender_enable = true
+```
+
 ## Enabling SSL
 
-Add the following parameters under the RabbitMQ configuration section to the deployment.toml file (stored in the `MI_HOME/conf` directory).
+Add the following parameters to enable SSL for the RabbitMQ listener.
+
+!!! Tip
+	  Note that keystore information is not required for an SSL connection if the <code>fail_if_no_peer_cert</code> parameter is set to 'false' in the RabbitMQ broker. You only need to enable SSL in the Micro Integrator (using the `parameter.ssl_enable` parameter shown below).
+
+    However, if the <code>fail_if_no_peer_cert</code> parameter is set to 'true' in RabbitMQ, the keystore configurations (given below) are also required for the Micro Integrator.
+
+    Shown below is a sample broker configuration where `fail_if_no_peer_cert` is set to `false`:
+    ```xml
+    {ssl_options, [{cacertfile,"/path/to/testca/cacert.pem"},
+                   {certfile,"/path/to/server/cert.pem"},
+                   {keyfile,"/path/to/server/key.pem"},
+                   {verify,verify_peer},
+                   {fail_if_no_peer_cert,false}]}   
+    ```
```

Comment:
```suggestion
    Shown below is an example of the config file where `fail_if_no_peer_cert` is set to `false`:
    ```
    ssl_options.cacertfile = /path/to/ca_certificate.pem
    ssl_options.certfile   = /path/to/server_certificate.pem
    ssl_options.keyfile    = /path/to/server_key.pem
    ssl_options.verify     = verify_peer
    ssl_options.fail_if_no_peer_cert = false
    ```
```